ban/http/resources.py

```diff
@@ -85,6 +85,14 @@ def get_collection(self, req, resp, **params):
     def on_get(self, req, resp, **params):
         """Get {resource} collection."""
         qs = self.get_collection(req, resp, **params)
+        for param in set(list(req.params)):
```

Comment:
This does not make sense actually: identifiers are unique fields, so filtering only through identifiers would mean only retrieving one. And we already can use the identifiers to get the resource through the resource endpoint.